package.json

```diff
@@ -16,7 +16,7 @@
     "predev": "npm run build",
     "dev": "http-server -o example/index.html -s -c-1 -p 8080",
     "build:css": "cleancss src/lettercrap.css --output dist/lettercrap.min.css",
-    "build:ts": "esbuild src/index.ts --bundle --minify --format=esm --tsconfig=tsconfig.json --outfile=dist/lettercrap.min.js",
+    "build:ts": "esbuild src/index.ts --bundle --minify --legal-comments=eof --format=esm --tsconfig=tsconfig.json --outfile=dist/lettercrap.min.js",
```

Comment:
According to the esbuild documentation, `legal-comments` is set to `eof` by default when using the `bundle` flag. We can keep it but it shouldn't be necessary.

Reply:
Yeah, you're right, that's an oversight on my part. I think I was using `/**` comments, it didn't work, and then I added the flag, only to realize that the comment has to begin with `/*!`.